File: python/7 kyu/278.py
# https://www.codewars.com/kata/586bca7fa44cfc833e00005c/train/python

# Passed

def create_array_of_tiers(n):
    result = []
    str_num = str(n)
    for end_idx in range(1, len(str_num) + 1):
        result.append(str_num[:end_idx])
        
    return result

output = create_array_of_tiers(80200)
print(output)

File: nodejs/7 kyu/278.js
// https://www.codewars.com/kata/555bfd6f9f9f52680f0000c5/train/javascript

// Passed

function reverseNumber(n) {
  let isNegative = n < 0;
  n = Math.abs(n);
  let reversedNum = Number(String(n).split("").reverse().join(""));
  if (isNegative) {
    reversedNum *= -1;
  }

  return reversedNum;
}

const output = reverseNumber(98989898);
console.log(output);